tests/containers/bci_test.pm

```diff
@@ -102,7 +102,7 @@ sub run {
     assert_script_run('source bci/bin/activate') if ($bci_virtualenv);
 
     record_info('Run', "Starting the tests for the following environments:\n$test_envs");
-    assert_script_run("cd /root/BCI-tests && git fetch && git reset --hard");
+    assert_script_run("cd /root/BCI-tests && git fetch && git reset --hard origin/main");
```

Comment:
Why not do `git pull --ff` instead of `git fetch && ...` ?

Reply:
Let me try this.

Reply:
I would recommend to use `get_var('BCI_TESTS_BRANCH', 'origin/main')`.

Reply:
Updated the PR.